meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -119,6 +119,8 @@ BBMASK += " \
     /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.110.bbappend \
     /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.113.bbappend \
     /meta-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-stm32mp_2.6.bb \
+    /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.115.bbappend \
+    /meta-st-stm32mp/recipes-devtools/gcc/gcc-source_12.2.bbappend \
```

Comment:
We can probably drop the first 2 lines (libdrm) and replace with the newer version instead.

Reply:
done